### PR TITLE
Version 1 Complete Iteration 1

### PR DESCRIPTION
**Pull request by mcat56**
@zacisaacson @freeheeling @sasloan 

Let me know if you catch anything!

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Looks awesome!!!

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  def test_attributes
+    assert_equal "game objects", @stat_tracker.game_objs
+    assert_equal "team objects", @stat_tracker.team_objs
+    assert_equal "game team objects", @stat_tracker.game_teams_objs
+  end
+
+  def test_from_csv
+    locations = { games: './data/dummy_games.csv', teams: './data/dummy_teams.csv', game_teams: './data/dummy_game_teams.csv' }
+    stat_tracker_2 = StatTracker.from_csv(locations)
```
Comment (freeheeling) [inline test/stat_tracker_test.rb:27]: Insert line.

File: test/teams_test.rb [hunk 2]
```diff
@@ -7,6 +7,11 @@ def setup
     @line_2 = Team.new("1,23,Atlanta United,ATL,Mercedes-Benz Stadium,/api/v1/teams/1")
   end
 
+  def test_it_exists
+    assert_instance_of Team, @line_2
+  end
+
+
```
Comment (freeheeling) [inline test/teams_test.rb:14]: Delete line.

**Comment by freeheeling** [comment]
Approved.

**Review by freeheeling — APPROVED** [review]
Approved.

File: test/stat_tracker_test.rb (hunk 1 above)
Comment (mcat56) [reply test/stat_tracker_test.rb:27]: Done! I will resubmit with my "pseudo code" for iteration 2

File: test/teams_test.rb (hunk 2 above)
Comment (mcat56) [reply test/teams_test.rb:14]: Good call, readability is important!

**Pull request merged**